src/main/java/com/wiley/autotest/selenium/elements/upgrade/Window.java

```diff
@@ -24,4 +26,14 @@
 
     void maximize();
 
+    void back();
+
+    void forward();
+
+    void refresh();
+
+    void to(String url);
```

Comment:
"to" sounds clear in WebDriver called after "navigate". What do you think if we rename "to" to "navigateTo"
or maybe  "openUrl"?

Reply:
gonna rename myself to add it faster ;)